mlfinlab/tests/test_trend_scanning_labels.py

```diff
@@ -23,9 +23,10 @@ def setUp(self):
         # Data set used for trend scanning labels
         self.eem_close = pd.read_csv(project_path + '/test_data/stock_prices.csv', index_col=0, parse_dates=[0])
         # In 2008, EEM had some clear trends
-        self.eem_close = self.eem_close['EEM'].loc[pd.Timestamp(2008, 4, 1):pd.Timestamp(2008, 10, 1)]
-
-    def test_trend_scanning_labels(self):
+        # self.eem_close = self.eem_close['EEM'].loc[pd.Timestamp(2008, 4, 1):pd.Timestamp(2008, 10, 1)]
```

Comment:
Can we drop this commented block of code?

Reply:
Yes, this can be deleted.